index.js

```diff
@@ -2,7 +2,6 @@
 
 'use strict';
 
-const exit = require('exit');
```

Comment:
No need to 'force exit', process will terminate when it has nothing left to do in event loop, and .exitCode can be set.